Makefile

```diff
@@ -47,3 +47,5 @@ hex_variation:
 
 images: squares hanoi intro escher_square lenet logo hilbert koch tensor latex hex_variation
 
+README.md: readme.py
+	jupytext --to ipynb --execute readme.py; jupyter nbconvert --to markdown --output README.md readme.ipynb
```

Comment:
Should this have been
```bash
jupytext --to ipynb --execute README.md
```
Or did you just forget to commit the `readme.py` file? Or am I misunderstanding something?